neptune/model.py

```diff
@@ -78,12 +78,10 @@ def system_properties(self):
             'created': entry.timeOfCreation,
             'finished': entry.timeOfCompletion,
             'running_time': entry.runningTime,
-            'source_code_size': entry.sourceSize,
             'owner': entry.owner,
             'size': entry.size,
             'tags': entry.tags,
-            'notes': entry.description,
-            'git_hash': entry.commitId
```

Comment:
Why was `sourceSize` and `commitId` removed?

Reply:
It was more complicated to obtain these two fields from backend api than it was from leaderboard, so we decided to drop it, at least for now.